source/payment_flow/index.html.md.erb

```diff
@@ -58,7 +59,7 @@ When your user reaches a page in your service where they need to make a payment,
 
 You should also have a clear call to action that tells your user they're about to pay. For example, a button that says __Pay now__ or __Continue to payment__.
 
-You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment.
+You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment. The following image shows an example payment page.
 
 <%= image_tag "/images/flow-service-payment-page.png", { :alt => '' } %>
```

Comment:
Do we need an ALT tag here?

Reply:
@iqbalgds If you explain the image in the main text, you should use alt=”” as this reduces auditory clutter with 
screen readers. I added the text "The following image shows an example payment page" to do this. The other images in this page aren't explicitly explained in the main text, and doing so would have meant more changes to the content than I think necessary.

Reply:
Thanks